src/test/resources/features/Opportunity.feature

```diff
@@ -1,7 +1,44 @@
 Feature: Opportunity
 
+
   Scenario: Change an owner
     Given I create opportunity as OpportunityTest
     When I change an opportunity's owner with "CurrentOwner"
     Then the application should display an information message in Opportunity page with the format "[[Owner Name]] now owns the record for [[Opportunity Name]]"
-    
\ No newline at end of file
+
+  @wip
+  Scenario: Close an opportunity with an another owner
+    Given I login as "Admin"
+    And I navigate to Opportunities Page
+    And I create an Opportunity with
+      | Opportunity Name | testing close stage |
+      | Close Date       | 03/31/2020          |
+      | Stage            | Qualification       |
+    And I select to "Juan Martinez" as a new owner
+    When I close the opportunity as close won
+    And I save it
+    Then the application should display an information message in Opportunity page with the format "Stage changed successfully."
+    And On the opportunities page Should display On the column stage as "Closed Won"
```

Comment:
It would be "the opportunities page should display  on the column stage as "Closed Won""